### ISSUE TITLE
Large PNG output

### ISSUE THREAD

**Issue by edreal** [lovell/sharp#2215]
This library is great but the only thing what is really frustrating is that the png files even when resized to equal size get around 30% larger.

I would like to try with the libimagequant and playing with the quality option but looks like most of those packages which are required for custom libvips build can't be installed in Debian 9 with apt-get.

**Comment by lovell** [comment]
The availability of `libimagequant-dev` from the `apt` package manager requires Debian 10 onwards.

https://packages.debian.org/buster/libimagequant-dev

This is a limitation of Debian rather than sharp, but please don't take out your frustration either here or there.

**Comment by edreal** [comment]
Alright I got it installed from their Github repo.

I compared the results with and without the palette option.

`await sharp('image.png').toFile('output.png');`
Output is 432 kb (took 134 ms)

`await sharp('image.png').png({palette: true}).toFile('output.png');`
Output is 92 kb (took 850 ms)

The source image used had original size of 333 kb.

Is it normal that the palette or image quant thing takes around 6x longer?
The output is significantly smaller though.

What was the reason why libimagequant wasn't added to the project and needs to be installed manually btw?
